app/jobs/geocode_job.rb

```diff
@@ -0,0 +1,9 @@
+class GeocodeJob < ActiveJob::Base
+  def perform(location)
```

Comment:
I haven't used ActiveJob yet -- are you able to pass instances through like that? In the past I had to pass IDs all over the place and reconstitute the object from that. This is a nice improvement if that is no longer necessary.